src/apple/templates.ts

```diff
@@ -0,0 +1,47 @@
+export function getRunScriptTemplate(
+    orgSlug: string,
+    projectSlug: string,
+    apiKey: string,
+    uploadSource = true,
+): string {
+    // eslint-disable-next-line no-useless-escape
+    return `# This script is responsable to upload debug symbols and source context for Sentry.\\nif which sentry-cli >/dev/null; then\\nexport SENTRY_ORG=${orgSlug}\\nexport SENTRY_PROJECT=${projectSlug}\\nexport SENTRY_AUTH_TOKEN=${apiKey}\\nERROR=$(sentry-cli debug-files upload ${uploadSource ? "--include-sources " : ""}\"$DWARF_DSYM_FOLDER_PATH\" 2>&1 >/dev/null)\\nif [ ! $? -eq 0 ]; then\\necho \"warning: sentry-cli - $ERROR\"\\nfi\\nelse\\necho \"warning: sentry-cli not installed, download from https://github.com/getsentry/sentry-cli/releases\"\\nfi\\n`;
+}
+
+export const scriptInputPath = "\"${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Resources/DWARF/${TARGET_NAME}\"";
+
+export function getSwiftSnippet(dsn: string): string {
+    return `           SentrySDK.start { options in
+            options.dsn = "${dsn}"
+            options.enableTracing = true
+            #if DEBUG
+            options.debug = true
+            options.environment = "Development"
+            #else
+            options.environment = "Release"
+            #endif
+            options.attachScreenshot = true
+            options.attachViewHierarchy = true
```

Comment:
`h`: I don't think we should enable that by default. Especially the screenshots could contain sensitive data, even in a development build.

Reply:
if I may, we could just add a comment to the snippet explaining that it can have sensitive data and turn it off in case you don't want it. Or we could add another step "Enable experimental features (could send PII to Sentry)"? and only after accepting it add the snippet

Reply:
I like the idea to show this options in the wizard and explain whats happening, but in another PR.

Reply:
I think adding a comment won't be sufficient. With the wizard, it can happen that some people don't check the code that much. They just hit compile and run after executing the wizard. We could add them //commented out or an extra step would also be OK.